src/main/java/mate/academy/exception/DataProcessingException.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.exception;
+
+public class DataProcessingException extends RuntimeException {
+    public DataProcessingException(String message) {
```

Comment:
do u use it anywhere ?

Reply:
Yes. 
[http://joxi.ru/823e9vEczMxONr](In MovieDaoImpl)
[http://joxi.ru/v29eLGoc49olW2](In MovieServiceImpl)

Reply:
Yes. MovieServiceImpl -> method public Movie get(Long id) and MovieDaoImpl -> method public Movie add(Movie movie)